doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -300,6 +300,56 @@ can be launched explicitly using :func:`create_launcher() <ansys.fluent.core.lau
   >>> pim_solver_session = pim_solver_launcher()
 
 
+Launching a Fluent Session in Container Mode Using Docker and Podman Compose
+----------------------------------------------------------------------------
+
+You can easily launch an Ansys Fluent session in container mode by leveraging Docker and Podman Compose with PyFluent. 
+This approach enables you to manage multi-container workflows and ensures a consistent, reproducible environment for your simulations.
+
+1. **Docker compose**
+
+    Prerequisites:
+
+    - `Docker <https://www.docker.com/>`_
+    - `Docker Compose <https://docs.docker.com/compose/>`_
+
+    Example:
+
+    .. code:: python
+
+      >>> import os
```

Comment:
@hpohekar @mkundu1 Is the use of environment variables here necessary? Do we expose the same capabilities programmatically? If not, should we not add that and promote it?

Reply:
yes, this is a good point 👍, since it is user facing feature.